packages/analytics/analytics-config-store/src/stores/analytics-config.ts

```diff
@@ -46,5 +45,6 @@ export const useAnalyticsConfigStore = defineStore('analytics-config', () => {
 
   return {
     getConfig,
+    configResult,
```

Comment:
As discussed in Slack, I still don't think that both of these should be exported.  If you feel strongly about not accessing the ref through a function, we could instead query the endpoint when the store is instantiated and get rid of `getConfig` entirely.